Source/Hiperion/Hiperion/Web.config

```diff
@@ -19,6 +19,8 @@
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
     <add key="owin:appStartup" value="Hiperion.Startup" />
+    <add key="FacebookAppId" value="463178923801598" />
+    <add key="FacebookAppSecret" value="dc333f2f5f885aeff5385feb56ad7006" />

```

Comment:
(I was just checking around)

You probably don't want to send this to the public repository, make sure to delete it so that nobody else can use it.